staging/src/k8s.io/api/apps/v1/types.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package v1
 
 import (
-	"k8s.io/api/core/v1"
```

Comment:
Please revert this file and I can approve this PR.

Reply:
done